lib/lolcommits/plugins/term_output.rb

```diff
@@ -0,0 +1,38 @@
+# -*- encoding : utf-8 -*-
+require 'rest_client'
+require 'base64'
+
+module Lolcommits
+  class TermOutput < Plugin
+
+    def initialize(runner)
+      super
+      options
+    end
+
+    def run_postcapture
+      return unless valid_configuration?
+
+      if !runner.vcs_info || runner.vcs_info.repo.empty?
+        puts 'Repo is empty, skipping output'
+      else
+        base64 = Base64.encode64(open(runner.main_image) { |io| io.read })
```

Comment:
Travis / Rubocop is complaining about style [here](https://travis-ci.org/mroth/lolcommits/jobs/147363210)